src/plugins/ui_feature/ui_sheet.ts

```diff
@@ -74,28 +74,39 @@ export class SheetUIPlugin extends UIPlugin {
   // Getters
   // ---------------------------------------------------------------------------
 
-  getCellWidth(position: CellPosition): number {
-    const text = this.getCellText(position);
+  getCellWidth(position: CellPosition): number | undefined {
```

Comment:
How can a cell have an "undefined" width ? It looks strange.

Reply:
For empty cells, but I can probably get away with returning 0